src/scripts/generate-binary-ops.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import argparse
+import settings
+import collections
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate
+import random
+import math
+
+def get_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--op", help = "give the operators to be tested", nargs = '+',
+    default = ["+", "-", "/", "*", "%", "^"], choices = ["+", "-", "/", "*", "%", "^"])
+    parser.add_argument("--test-count", help = "give the number of tests to run", nargs = 1, default = [500],
+    type = int)
+    parser.add_argument("--output", help = "give the path to generate files into", default = settings.NUMBER_DIR)
+    parser.add_argument("--operand-count", help = "give the number of maximum operands in an expression",
+    nargs = 1, default = [10], choices = range(2, 11), type = int)
+    parser.add_argument("--seed", help = "give the random seed value", nargs = 1, default = [10000],
+    type = int)
+    if len(sys.argv) == 1:
+        parser.print_help()
+        sys.exit(1)
+
+    script_args = parser.parse_args()
+
+    return script_args
+
+filename = '''binary-ops-{NUMBER}.js'''
+MAX = (1<<53) - 1
+MIN = -1 * MAX
+BITMAX = 9999999
+BITMIN = -9999999
```

Comment:
`BITMIN = -BITMAX`

Reply:
Where do these numbers come from? Please add a description for all of them.

Reply:
I changed some of them and added a description for constants and variables.